packages/@aws-cdk/cx-api/lib/cxapi.ts

```diff
@@ -26,8 +26,17 @@ export type CXRequest = ListStacksRequest | SynthesizeRequest;
  */
 export interface MissingContext {
   provider: string;
-  scope: string[];
-  args: string[];
+  props: {
+    account: string;
```

Comment:
I believe `account` and `region` are still potentially `undefined` (for providers that don't require them--even though we don't have any right now), so add a `?` to them.